docs/quick-start.mdx

```diff
@@ -13,58 +13,46 @@ Uppy is a sleek, modular JavaScript file uploader that integrates seamlessly
 with any application. It’s fast, has a comprehensible API and lets you worry
 about more important problems than building a file uploader.
 
-- **Fetch** files from local disk, remote URLs, Google Drive, Dropbox, Box,
-  Instagram or snap and record selfies with a camera
-- **Preview** and edit metadata with a nice interface
-- **Upload** to the final destination, optionally process/encode
-
-## Features
-
-- Lightweight, modular plugin-based architecture, light on dependencies :zap:
-- Resumable file uploads via the open [tus](https://tus.io/) standard, so large
-  uploads survive network hiccups
-- Supports picking files from: Webcam, Dropbox, Box, Google Drive, Instagram,
-  bypassing the user’s device where possible, syncing between servers directly
-  via [@uppy/companion](/docs/companion)
-- Works great with file encoding and processing backends, such as
-  [Transloadit](https://transloadit.com), works great without (all you need is
-  to roll your own Apache/Nginx/Node/FFmpeg/etc backend)
-- Sleek user interface :sparkles:
-- Optional file recovery (after a browser crash) with
-  [Golden Retriever](/docs/golden-retriever/)
-- Speaks several languages (i18n) :earth_africa:
-- Built with accessibility in mind
-- Free for the world, forever (as in beer 🍺, pizza 🍕, and liberty 🗽)
-- Cute as a puppy, also accepts cat pictures :dog:
-
 :::tip
 
-You can take Uppy for a walk inside
-[CodeSandbox](https://codesandbox.io/s/uppy-dashboard-xpxuhd).
+You can take Uppy for a walk inside CodeSandbox with a
+[minimal drag & drop](https://codesandbox.io/s/uppy-drag-drop-gyewzx?file=/src/index.js)
+experience or a
+[full featured dashboard](https://codesandbox.io/s/uppy-dashboard-xpxuhd).
 
 :::
 
 <QuickStartLinks
 	items={[
 		{
-			name: 'Examples',
-			description: 'Try out one of our extensive example projects',
-			link: 'https://github.com/transloadit/uppy/tree/main/examples',
+			name: 'I want a full featured, extendable dashboard',
```

Comment:
```suggestion
			name: 'I want a full featured, extendable UI with drag-drop, file previews and upload progress',
```

Reply:
I need to keep the sentences short or it doesn't look as good / less readable

<img width="1042" alt="Screenshot 2023-03-22 at 13 57 08" src="https://user-images.githubusercontent.com/9060226/226911594-138bf397-fc64-4793-975c-b0893c48994e.png">